packages/storybook/.storybook/shared/config.js

```diff
@@ -1,6 +1,9 @@
 import { addDecorator } from "@storybook/react";
-import { withInfo } from "@storybook/addon-info";
 import { withKnobs } from "@storybook/addon-knobs/react";
+import { setDefaults } from '@storybook/addon-info';
 
-addDecorator((story, context) => withInfo({ header: true })(story)(context));
```

Comment:
@morrisallison https://github.com/Autodesk/hig/pull/833#discussion_r179559616 that day is today!